Samples/StorageProviders/StorageProviders/BaseJSONStorageProvider.cs

```diff
@@ -15,6 +17,13 @@ namespace Samples.StorageProviders
     /// </summary>
     public abstract class BaseJSONStorageProvider : IStorageProvider
```

Comment:
just have in mind that we are talking about the Tester project, which is where the duplicate files live. They are not in the main providers that we ship. That's why I'm suggesting using linked files.
Still, I'm advocating that the MongoDb provider should NOT be included in the Tester project (currently it isn't, so keeping it as is, just in the Samples).